ed25519/options.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package ed25519
+
+import (
+	"crypto/ed25519"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"strings"
+
+	wrapping "github.com/hashicorp/go-kms-wrapping/v2"
+)
+
+const (
+	ConfigPrivKey     = "priv-key"
```

Comment:
See e.g. https://github.com/hashicorp/go-kms-wrapping/blob/main/aead/options.go#L53 -- we tend to use underscores, not hyphens.

Reply:
ty